### PR TITLE
fix: stats not implemented on jsipfs

### PR DESCRIPTION
**Pull request by hacdias**
Ref.: https://github.com/ipfs/js-ipfs/pull/1189

### CODE REVIEW

**Comment by hacdias** [comment]
@diasdavid could you check if [this](https://github.com/ipfs/interface-ipfs-core/pull/209/files#diff-6635c0bbe87eda053376dbd64f1dab6cR101) (only returning when we're using promises) is right?

**Comment by hacdias** [comment]
@diasdavid just confirmed and it works.

File: src/stats.js [hunk 1]
```diff
@@ -76,7 +96,12 @@ module.exports = (common) => {
     })
 
     it('.bw Promise', () => {
-      return ipfs.stats.bw().then((res) => {
+      if (!withGo) {
+        console.log('Not supported in js-ipfs yet')
+        return
+      }
+
+      ipfs.stats.bw().then((res) => {
```
Comment (diasdavid) [inline src/stats.js]: @hacdias you should add a return here. Otherwise the test will pass before the call happens.

Comment (hacdias) [reply src/stats.js]: OOPS! Sorry. fixed!

**Comment by hacdias** [comment]
@diasdavid fixed 😄

File: src/stats.js [hunk 2]
```diff
@@ -101,7 +101,7 @@ module.exports = (common) => {
         return
       }
 
-      ipfs.stats.bw().then((res) => {
+      return ipfs.stats.bw().then((res) => {
```
Comment (diasdavid) [inline src/stats.js]: @hacdias have you checked if this is done in all the tests?

Comment (hacdias) [reply src/stats.js]: Yes, on all promises tests

Comment (diasdavid) [reply src/stats.js]: thank you 👍

**Pull request merged**